ElDorito/Source/Discord/DiscordRPC.cpp

```diff
@@ -15,29 +20,52 @@
 
 #include "../Utils/Logger.hpp"
 
+//This applicationID is owned by Luke#4157 . Please DM if you need an image added.
+static const char* APPLICATION_ID = "378684431830876170";
```

Comment:
We'd like to use the application we created. Please message me about what we can do there. Maximumgame#7961